x-pack/legacy/plugins/infra/types/eui.d.ts

```diff
@@ -96,6 +96,7 @@ declare module '@elastic/eui' {
     message?: any;
     rowProps?: any;
     cellProps?: any;
+    responsive?: boolean;
```

Comment:
:information_source: This was just used somewhere, and I honestly have no idea how this didn't fail in the current TS .